gui/default/syncthing/core/syncthingController.js

```diff
@@ -53,27 +53,30 @@ angular.module('syncthing.core')
         $scope.themes = [];
         $scope.globalChangeEvents = {};
         $scope.metricRates = false;
+        $scope.fsNotificationsAvailable = undefined;
```

Comment:
So it's already undefined, we should probably removed the other undefined things to avoid confusion.